app/views/transferConfirmation.scala.html

```diff
@@ -1,12 +1,24 @@
-@()(implicit messages: Messages)
+@(exportTriggered: Boolean)(implicit messages: Messages)
 @main(Messages("transferConfirmation.header")) {
     @defining(play.core.PlayVersion.current) { version =>
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-                <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
-                @progressIndicator(Messages("transferConfirmation.progress"))
-                <p>Placeholder page</p>
+        @if(exportTriggered) {
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+                    @progressIndicator(Messages("transferConfirmation.progress"))
+                    <p>Export complete placeholder page</p>
+                </div>
             </div>
-        </div>
+        } else {
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <span id="general-error" class="govuk-error-message">
+                        The export failed, contact us
```

Comment:
Is there a card for returning to this page and handling both conditions?

Reply:
I couldn't see one so I've added in TDR-672